src/Hill.jl

```diff
@@ -138,7 +138,7 @@ end
 
 """ Calculate the # of cells in G1 for a set of parameters and T """
 function numcells(params, g0, T)
-    @assert(all(params .>= 0.0), "negative params $params")
+#     @assert(all(params .>= 0.0), "negative params $params")
```

Comment:
Why comment this out?

Reply:
Oops, sorry forgot to uncomment, although I have another assert in the BlissCombination that does the same.